modules/order/src/Form/MultipleProfileTypesConfirmForm.php

```diff
@@ -355,10 +358,12 @@ protected function getExistingOrderCountDescription() {
   protected function getIncompatibleModules() {
     $incompatible_modules = [];
 
+    // TODO: We don't know yet which module version will be supporting multiple
+    // profile types yet. Setting to empty for now.
     $affected_modules = [
-      'commerce_pos' => '2.2',
-      'commerce_shipping' => '2.2',
-      'commerce_amws' => '2.2',
+      'commerce_pos' => '',
+      'commerce_shipping' => '',
+      'commerce_amws' => '',
```

Comment:
If there is no known compatible version, then the module should be marked as incompatible. Right now it seems that it is accepted.

Reply:
Fixed.